python/README.md

```diff
@@ -0,0 +1,25 @@
+# Python Arrow Flight Client Exmaple
+This lightweight Python client connects to the Dremio Arrow Flight service endpoint. This client requires the username and password for authentication. The hostname and port used is default to `localhost` and `32010`, these settings can be changed by providing the hostname and port as arguments when running the client. Moreover, the tls option can be provided to establish an encrypted connection. Please note that trusted certificates must be provided when the tls option is enabled. This client can query any datasets in Dremio that are accessible by the Dremio user provided.
+
+## Instructions on using this Python client
+- Install and setup Python3 as `pyarrow` requires Python3
+- Install `pyarrow` package using pip
+  - `pip3 install pyarrow`
+- Install `pandas` package using pip
```

Comment:
probably better to add a `requirements.txt` with these packages pinned: 

```
pyarrow==0.2.0
pandas==<current verison of pandas>
```

Then the instructions are just `pip install -r requirements.txt`

Reply:
also worth considering `conda` instead of straight python. or at least alternative instructions. pyarrow isn't built for all python/os combos and will result in a MASSIVE compilation if their platform isn't supported

Reply:
Modified installation instructions.